test/integration/imu_system.cc

```diff
@@ -212,3 +212,43 @@ TEST_F(ImuTest, ModelFalling)
   EXPECT_EQ(imuMsgs.back().entity_name(), scopedName);
   mutex.unlock();
 }
+
+/////////////////////////////////////////////////
+// The test checks to make sure orientation is not published if it is deabled
+TEST_F(ImuTest, OrientationDisabled)
+{
+  imuMsgs.clear();
+
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/test/worlds/imu_no_orientation.sdf";
+  serverConfig.SetSdfFile(sdfFile);
+
+  Server server(serverConfig);
+  EXPECT_FALSE(server.Running());
+  EXPECT_FALSE(*server.Running(0));
+
+  const std::string sensorName = "imu_sensor";
+
+  auto topic =
+      "world/imu_sensor/model/imu_model/link/link/sensor/imu_sensor/imu";
```

Comment:
common::joinPaths

Reply:
hmm this string if for topic name so I think it's not necessary to use `joinPaths` as that's intended for file paths?